### PR TITLE
ISO/OVA closure size reductions

### PR DESCRIPTION
**Pull request by matthewbauer**
###### Motivation for this change

A few things to partially address #47292. Gets rid of GCC and Qt4 from the .ova build products (and GCC from the .iso)

- Properly strips nfs-utils binaries
- Disables Qt4 in the graphical profile (pretty soon we should make this the default everywhere)

### CODE REVIEW

**Comment by xeji** [comment]
@GrahamcOfBorg test nfs3
@GrahamcOfBorg test nfs4

File: pkgs/os-specific/linux/nfs-utils/default.nix [hunk 1]
```diff
@@ -83,6 +83,8 @@ in stdenv.mkDerivation rec {
     "statdpath=$(TMPDIR)"
   ];
 
+  stripDebugList = [ "lib" "libexec" "bin" "etc/systemd/system-generators" ];
+
```
Comment (vcunat) [inline pkgs/os-specific/linux/nfs-utils/default.nix]: Compiled binaries in `etc/**` :scream:

**Comment by vcunat** [comment]
According to Hydra the graphical ISO has exactly the same size and OVA won't evaluate.
```
in job ‘nixos.ova.x86_64-linux’:
attribute 'diskInterface' missing, at /nix/store/5k9kk52bv6zsvsyyvpxhm8xmwyn2yjvx-source/pkgs/build-support/vm/default.nix:316:24
```

**Comment by xeji** [comment]
The OVA eval error is unrelated, it was caused by #34409 which was reverted a few hours ago.

**Comment by vcunat** [comment]
Right.  Hydra shows the OVA size did go down by roughly 25 MiB.

**Pull request merged**